pages/phases/assessments/written-assessments.md

```diff
@@ -6,21 +6,17 @@ sidenav: hiring-phases
 phase: assessments
 sub-phase-order: 020
 title: Written Assessments
-intro: During job analysis, SMEs and HR may decide to do a written assessment or demonstration project instead of a first-round interview.
+intro: During job analysis, SMEs and HR may decide to do a written assessment instead of a first-round interview.
 ---
 
 <p class="usa-intro">
   {{ page.intro }}
 </p>
 
-If your SMEs opt for a written assessment or demonstration project instead of a first-round interview, you will need to prepare questions and training for each different assessment round.
+If your SMEs opt for a written assessment instead of a first-round interview, you will need to prepare questions and training for each different assessment round.
 
 The assessment type should be appropriate to evaluate the applicant's proficiency level within the examined competencies.
 
 ### Assessment Options
 
-- Demonstration Projects
-- Written Prompts
-- Structured Interviews
-
-See a longer list at [this reference link](https://www.opm.gov/policy-data-oversight/assessment-and-selection/other-assessment-methods/).
+See a full list at [this reference link](https://www.opm.gov/policy-data-oversight/assessment-and-selection/other-assessment-methods/).
```

Comment:
Will I changed this again....i cant stand structured vs non structured essay bc i dont even know what that means...

Reply:
Can we just say "essay" then?